docs/lib/storage/fragments/ios/configureaccess.md

```diff
@@ -1,19 +1,26 @@
 ## Protected access
 
-Create an options object with the protected access level to restrict access for certain objects.
+Create an `Options` object with the `.protected` access level to restrict access for your key.
```

Comment:
Niggle: The key refers to the location of an object, or, to be storage system agnostic, a file. What we're restricting access to is the file or object, not the key.